docs/extensions/build.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Build
```

Comment:
Looks good although we should add the Python dependency that was identified by a customer recently.

Reply:
do you recall what it was? let me know, and i can add it- or you can suggest it and we can commit directly.

Reply:
A customer could not install 0.8.0 in a Python 3.11 environment. We can add this in a later PR